test/e2e/iaa/iaa.go

```diff
@@ -122,25 +123,7 @@ func describe() {
 
 	ginkgo.Describe("With using operator", func() {
```

Comment:
how about moving these to the operator module? It would allow to deploy the operator only once and then have separate steps for each device.

Reply:
@mythi done!